tests/unit/protocol.tcl

```diff
@@ -69,37 +69,37 @@ start_server {tags {"protocol network"}} {
     }
 
     set c 0
-    foreach seq [list "\x00" "*\x00" "$\x00"] {
+    foreach seq [list "\x00" "*\x00" "$\x00" "*1\r\n$\x00"] {
```

Comment:
This adds some new thing to test? Not only fixing a failing test it seems.

Reply:
right - this was not covered by the early test